dbms/src/Flash/FlashService.cpp

```diff
@@ -235,10 +238,60 @@ grpc::Status FlashService::IsAlive(grpc::ServerContext * grpc_context [[maybe_un
     return grpc::Status::OK;
 }
 
-std::variant<grpc::Status, std::string> FlashService::establishMPPConnectionSyncOrAsync(grpc::ServerContext * grpc_context,
-                                                                                        const mpp::EstablishMPPConnectionRequest * request,
-                                                                                        grpc::ServerWriter<mpp::MPPDataPacket> * sync_writer,
-                                                                                        IAsyncCallData * call_data)
+std::variant<grpc::Status, std::string> AsyncFlashService::establishMPPConnectionAsync(grpc::ServerContext * grpc_context,
```

Comment:
I think this function can be removed. We can use `EstablishCallData::tryConnectTunnel` to do all these work.

Reply:
`establishMPPConnectionAsync ` need to do some validation check, I refine the code so `establishMPPConnectionAsync` will call `EstablishCallData::tryConnectTunnel`